packages/web/src/common/store/cache/collections/sagas.js

```diff
@@ -213,11 +234,13 @@ function* confirmCreatePlaylist(uid, userId, formFields, source) {
             }
           ])
         )
-        yield put(
-          cacheActions.update(Kind.COLLECTIONS, [
-            { id: uid, metadata: { _moved: subscribedUid } }
-          ])
-        )
+        if (!playlistEntityManagerIsEnabled) {
+          yield put(
+            cacheActions.update(Kind.COLLECTIONS, [
+              { id: uid, metadata: { _moved: subscribedUid } }
```

Comment:
do we need any of this `_moved` logic?

Reply:
i think it's still necessary for the entity manager disabled case. just trying to keep it the same as it was before.